### PR TITLE
Create default mntnr tables

### PR DESCRIPTION
**Pull request by maggarwal13**

### CODE REVIEW

**Review by petchells — APPROVED** [review]

File: whois-commons/src/main/resources/internals_schema.sql [hunk 1]
```diff
@@ -71,6 +71,28 @@ CREATE TABLE `default_maintainer_sync_history` (
     `is_synchronised` tinyint(1) DEFAULT 0,PRIMARY KEY (`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 
+
+DROP TABLE IF EXISTS `default_maintainer`;
```
Comment (eshryane) [inline whois-commons/src/main/resources/internals_schema.sql:75]: Is this the "in progress " table? Can it be named `default_maintainer_in_progress` to be explicit?

File: whois-commons/src/main/resources/internals_schema.sql [hunk 2]
```diff
@@ -71,6 +71,28 @@ CREATE TABLE `default_maintainer_sync_history` (
...
+    `id` int(10) UNSIGNED NOT NULL AUTO_INCREMENT,
+    `org` varchar(256) NOT NULL,
+    `mntner` varchar(256) NOT NULL,
+    `timestamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+    `uuid` varchar(256) NOT NULL,
+    `email` varchar(256),
+    PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+DROP TABLE IF EXISTS `default_maintainer_sync`;
```
Comment (eshryane) [inline whois-commons/src/main/resources/internals_schema.sql:86]: We need to script the pre-population of the sync table from the sync_history table

Comment (maggarwal13) [reply whois-commons/src/main/resources/internals_schema.sql:86]: Yes there are insert statements in a patch file with this PR

Comment (eshryane) [reply whois-commons/src/main/resources/internals_schema.sql:88]: Thanks

**Review by eshryane — APPROVED** [review]

**Pull request merged**